src/main/java/pl/javastart/task/Memory.java

```diff
@@ -0,0 +1,34 @@
+package pl.javastart.task;
+
+import java.util.InputMismatchException;
+
+public class Memory extends Processor {
```

Comment:
Dziedziczenie oznacza relację "jest", np. "pies jest zwierzęciem". Tutaj nie powinnaś stosować dziedziczenia, bo pamięć nie jest procesorem. Jeżeli masz wspólne cechy między procesorem i pamięcią związane z podkręcaniem, to możesz wprowadzić dodatkową klasę pośrednią (może być abstrakcyjna) typu OverclockableComponent i po niej mogłyby dziedziczyć Processor i Memory